programs/bpf/rust/invoke/src/lib.rs

```diff
@@ -45,6 +47,8 @@ const DERIVED_KEY2_INDEX: usize = 7;
 const DERIVED_KEY3_INDEX: usize = 8;
 const SYSTEM_PROGRAM_INDEX: usize = 9;
 const FROM_INDEX: usize = 10;
+const ED25519_PROGRAM_INDEX: usize = 11;
+// const INVOKE_PROGRAM_INDEX: usize = 12;
```

Comment:
how about being explicit of the reason of commenting-out?:

`// const INVOKE_PROGRAM_INDEX: usize = 12; // not used currently` 

Otherwise, it's a bit hard to tell if this is accidental (like some debugging leftover) or intended addition..